### PR TITLE
Fix: Upgrade chip dropdown is presented under the results modal [ED-15377]

### PR DESCRIPTION
**Pull request by DennisNerush**

### CODE REVIEW

File: modules/ai/assets/js/editor/components/upgrade-chip.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ const UpgradeChip = ( {
 			<Popper
 				open={ isPopoverOpen }
 				anchorEl={ anchorEl.current }
-				sx={ { zIndex: '9999', maxWidth: 300 } }
+				sx={ { zIndex: '170001', maxWidth: 300 } }
```
Comment (DennisNerush) [inline modules/ai/assets/js/editor/components/upgrade-chip.js:116]: above the AI modal zIndex

File: modules/ai/assets/js/editor/hooks/use-upgrade-message.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import useIntroduction from './use-introduction';
 
-export const USAGE_PERCENTAGE_THRESHOLD = 80;
+export const USAGE_PERCENTAGE_THRESHOLD = 75;
```
Comment (DennisNerush) [inline modules/ai/assets/js/editor/hooks/use-upgrade-message.js:3]: Per Shilo's request

File: modules/ai/assets/js/editor/hooks/use-upgrade-message.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ const useUpgradeMessage = ( { usagePercentage, hasSubscription } ) => {
 	const isBelowThreshold = usagePercentage < USAGE_PERCENTAGE_THRESHOLD;
 	const isAboveThreshold = usagePercentage >= USAGE_PERCENTAGE_THRESHOLD;
 
-	const showBadge = isAboveThreshold || ( isFreeUser && isBannerViewed );
+	const showBadge = isAboveThreshold || isFreeUser;
```
Comment (DennisNerush) [inline modules/ai/assets/js/editor/hooks/use-upgrade-message.js:12]: Upgrade badge should always be shown to free users

Comment (naorsabag) [reply modules/ai/assets/js/editor/hooks/use-upgrade-message.js:12]: but if they press the close button?

Comment (DennisNerush) [reply modules/ai/assets/js/editor/hooks/use-upgrade-message.js:12]: it's the badge, no close button on badge

Comment (naorsabag) [reply modules/ai/assets/js/editor/hooks/use-upgrade-message.js:12]: If you checked it then all fine, cause Its not what Im seeing..

**Review by naorsabag — DISMISSED** [review]

File: modules/ai/assets/js/editor/hooks/use-upgrade-message.js (hunk 3 above)
Comment (DennisNerush) [reply modules/ai/assets/js/editor/hooks/use-upgrade-message.js:12]: image
what are you seeing?

Comment (naorsabag) [reply modules/ai/assets/js/editor/hooks/use-upgrade-message.js:12]: yup but it is also for the upgrade message I think.. try to display the purple upgrade message and then try to close it and reopen the popup and see if it still there

**Review by naorsabag — APPROVED** [review]

**Pull request merged**